Bugsnag/Delivery/BugsnagErrorReportApiClient.m

```diff
@@ -30,16 +30,16 @@ - (void)main {
     @autoreleasepool {
         @try {
             [[BSG_KSCrash sharedInstance]
-                    sendAllReportsWithCompletion:^(NSUInteger sentReportCount,
+                    sendAllReportsWithBlock:^(NSString *filename,
                             BOOL completed, NSError *error) {
                         if (error) {
-                            bsg_log_warn(@"Failed to send reports: %@", error);
-                        } else if (sentReportCount > 0) {
-                            bsg_log_info(@"Reports sent.");
+                            bsg_log_warn(@"Failed to send error report: %@ %@", filename, error);
+                        } else if (filename) {
+                            bsg_log_info(@"Delivered error report '%@'", filename);
                         }
                     }];
```

Comment:
Do we need this block? It's just logging the outcome which is also done by `BSG_KSCrash.sendAllReportsWithBlock`

Reply:
This doesn't seem to be necessary so I've removed it. In addition this means `BSG_KSCrashCallCompletion` is no longer required so I've removed these files from the project.